### DISCUSSION TITLE
Have the money currency written instead of just having the symbol

### DISCUSSION THREAD

**Discussion by growtoups** [Paymenter/Paymenter#475] [Ideas]
Hello i thought that it would be great if you guys added the whole currency name like $CAD $USD $AUD and etc since its hard for customers to know the currency and can drive away those customers.

**Comment by CorwinDev** [comment]
![image](https://github.com/Paymenter/Paymenter/assets/88144943/08da8508-f632-45fe-922a-e824e9eef7a4)
V1 (coming in a few months) will support setting multiple currencies and prefix/suffix of it :)

**Reply by growtoups** [reply]
Alright thanks for the information!
